ios-common-build.gradle

```diff
@@ -0,0 +1,168 @@
+task cloneIOSSDK {
+    doLast() {
+        // clone iOS SDK at specified version, if it doesn't already exist
+        println("Downloading iOS SDK version ${airshipProperties.iosVersion}")
+        def tmpDir = System.getProperty("java.io.tmpdir")
+        def iosTempRepo = new File("${tmpDir}/ios-library-for-xamboni")
```

Comment:
This mostly seems like it should work, but it's a little weird to me that we're both cloning the repo directly into a temp directory with git, for this task, and using carthage for the frameworks. Is there a reason we need both? Could this potentially be simplified by using the carthage resources that the rest of the iOS gradle scripts depend on?

Reply:
That's is an excellent idea. I completely forgot that the Xamarin build uses Carthage. I'll take a look at it.